packages/local-signing-manager/src/lib/local-signing-manager.ts

```diff
@@ -190,11 +190,23 @@ export class LocalSigningManager implements SigningManager {
     let address: string;
 
     if ('uri' in account) {
-      ({ address } = keyring.addFromUri(account.uri));
+      ({ address } = keyring.addFromUri(
```

Comment:
(style) I think we should use two lines here to add readbility. Either assign on another line, or the derivation ternerary.

Reply:
Sure, resolved.